client/rest-high-level/src/main/java/org/elasticsearch/client/security/GetUserPrivilegesRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.security;
+
+import org.apache.http.client.methods.HttpGet;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.Validatable;
+
+/**
+ * A request object for the {@link org.elasticsearch.client.SecurityClient#getUserPrivileges(RequestOptions)} API.
+ * This request takes no parameters, and has a singleton {@link #INSTANCE}.
+ */
+public class GetUserPrivilegesRequest implements Validatable {
+
+    public static final GetUserPrivilegesRequest INSTANCE = new GetUserPrivilegesRequest();
+
+    private final Request request;
+
+    private GetUserPrivilegesRequest() {
+        request = new Request(HttpGet.METHOD_NAME, "/_xpack/security/user/_privileges");
```

Comment:
Since you're out tomorrow, #36293 will be merged before hand so `/_xpack/security/user/_privileges` -> `/_security/user/_privileges`

Reply:
We'll see who merges first :) Probably you, but I can't update this just yet.

Reply:
I was _TOO_ optimistic.